website/src/main/java/com/google/step/servlets/LoginServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.google.sps.servlets;
```

Comment:
I renamed sps (software product sprint) to the step folder, please update package name accordingly
```suggestion
package com.google.step.servlets;
```